### DISCUSSION TITLE
Some images not displaying

### DISCUSSION THREAD

**Discussion by hwlong0305** [CollectionBuilder/collectionbuilder.github.io#118] [General]
I am planning to demonstrate collectionbuilder in a library staff meeting tomorrow and have created a demo collection here: https://hwlong0305.github.io/exhibit-demo/ . Some of the images are not showing up. I've checked the size and they are all below the upper threshold. They are all jpgs. None of the archived posts about images not displaying seemed to be relevant to my situation. Any thoughts about why the images aren't displaying? Thanks in advance for any tips you can offer.

**Comment by dcnb** [comment]
hi @hwlong0305, 

I think the images I've checked just aren't in the [/objects/](https://github.com/hwlong0305/exhibit-demo/tree/main/objects) folder. There are 17 images in the folder from you (with some in there from the demo collection automatically included) and your metadata points to 28 items. So I think you just need to find those 11 that are missing and upload them!

If it's something more nefarious than this, just let us know!
